sdk/storage/storage-file-datalake/package.json

```
@@ -120,7 +120,7 @@
     "@azure/abort-controller": "^1.0.0",
     "@azure/core-http": "^2.0.0",
     "@azure/core-paging": "^1.1.1",
-    "@azure/core-tracing": "1.0.0-preview.13",
+    "@azure/core-tracing": "1.0.0",
```

Perhaps there is no need to pin anymore that it has been stable?
```suggestion
    "@azure/core-tracing": "^1.0.0",
```